### DISCUSSION TITLE
mongo: if v had a vlib/mongo what would you like the syntax to be.

### DISCUSSION THREAD

**Discussion by enghitalo** [vlang/v#15148] [General]

**Comment by JalonSolov** [comment]
https://github.com/vlang/mongo ?

**Reply by enghitalo** [reply]
Thank you!!!!
Why it is not in vliv/mongo, tho?

**Reply by JalonSolov** [reply]
Because vlib is supposed to be "lean & mean", with only the most used options.  Other things go in "external" modules.
